client/src/components/Query/Filter/index.jsx

```diff
@@ -112,7 +112,8 @@ class Filter extends React.Component {
   handleApply() {
     if (this.isEditable()) {
       const { draft } = this.state;
-      const { onEditCallback } = this.props;
+      const { editor , onEditCallback } = this.props
+      draft.values  = editor.props.children[6].props.children.props.children.props.value;
```

Comment:
@claudia1296 We will need to revise and find a better way to do this, but for now it works.